tonic/models/vic/grid_params.py

```diff
@@ -991,8 +1353,46 @@ def snow(snow_file, soil_dict, c=Cols(snow_bands=5)):
 
 
 # -------------------------------------------------------------------- #
-def veg(veg_file, soil_dict, max_roots=3, veg_classes=11,
-        cells=False, blowing_snow=False, lai_index=False):
+def veg_class(vegl_file, c=Cols(veglib_fcan=False)):
+    """
+    Load the entire vegetation library file into a dictionary of lists.
+    """
+
+    print('reading {0}'.format(vegl_file))
+
+    data = []
+    sep = ' '
```

Comment:
I think you're misunderstanding how I'm using "sep".  I'm using it as my own separator with which to *join* the various words in the comment into a single string.  So I'm decreeing that these words will be joined by a space.